src/template/mod.rs

```diff
@@ -45,4 +45,15 @@ macro_rules! solution {
             run_part(part_two, &input, DAY, 2);
         }
     };
+    ($day:expr, $part:expr) => { /// Allows testing partially implemented solutions on full input
+
+        const DAY: advent_of_code::Day = advent_of_code::day!($day);
+        const PART: u8 = $part;
+
+        fn main() {
+            use advent_of_code::template::runner::*;
+            let input = advent_of_code::template::read_file("inputs", DAY);
+            run_part(part_one, &input, DAY, PART);
```

Comment:
this is evaluating `part_one` regardless of what is passed in as `$part`. How would this work if I wanted to test `part 2` in isolation?

Reply:
Ah...valid point. 

Updated with literals for each case!